src/renderer/features/multisig-wallet/components/WalletRow.tsx

```diff
@@ -19,22 +14,9 @@ type Props = {
   onSelect: (wallet: Wallet) => unknown;
 };
 export const WalletRow = ({ wallet, onSelect }: Props) => {
```

Comment:
This component can be safely deleted, but it's `html` should be moved to `WalletGroup`